src/React.re

```diff
@@ -27,6 +30,19 @@ module Ref = {
 
 [@bs.module "react"] external createRef: unit => Ref.t(Js.nullable('a)) = "";
 
+module Children = {
+  [@bs.module "react"] [@bs.scope "Children"] [@bs.val]
+  external map: (element, element => element) => array(element) = "map";
```

Comment:
Why does this return array instead of element? If there’s only one child does it wrap in an array?

Reply:
yeah, it does. there's a special case with `null` and `undefined` though, might be better to treat the return value as `element`.